rastervision/backend/torch_utils/chip_classification/data.py

```diff
@@ -0,0 +1,99 @@
+from os.path import join
+
+import logging
+log = logging.getLogger(__name__)
+
+from torchvision.datasets import ImageFolder
+from torchvision.transforms import Compose, ToTensor
+from torch.utils.data import DataLoader
+
+from albumentations import (
+    HorizontalFlip,
+    VerticalFlip,
+    Transpose,
+    Rotate,
+    GaussNoise,
+    RandomGamma,
+    HueSaturationValue,
+    RGBShift,
+    RandomBrightness,
+    RandomContrast,
+    ChannelShuffle,
+    InvertImg,
+    ToGray,
+    RandomSnow,
+    RandomFog,
+    ChannelDropout
+)
+
+from rastervision.backend.torch_utils.data import DataBunch
+
+def build_databunch(data_dir, img_sz, batch_sz, class_names, augmentors):
+    num_workers = 4
+
+    train_dir = join(data_dir, 'train')
+    valid_dir = join(data_dir, 'valid')
+
+    aug_transform = [ToTensor()]
+    transform = [ToTensor()]
+
+    for augmentor in augmentors:
+        if augmentor == 'RandomHorizontalFlip':
+            aug_transform.append(HorizontalFlip(p=0.5))
```

Comment:
I thought the ToTensor() needs to come at the end of the list of transforms but I could be wrong.

Reply:
I think you are right. I have changed it. I'll test the debug chips tomorrow.